mainapp/views.py

```diff
@@ -501,18 +501,6 @@ def __init__(self, *args, **kwargs):
         if self.data == {}:
             self.queryset = self.queryset.none()
 
-class CoordinatorCampFilter(django_filters.FilterSet):
```

Comment:
did you remove these lines by mistake?

Reply:
code repetition  https://github.com/IEEEKeralaSection/rescuekerala/blob/master/mainapp/views.py#L690